src/nvim/api/private/helpers.c

```diff
@@ -1039,6 +1039,45 @@ Object copy_object(Object obj)
   }
 }
 
+int get_option_value_for(char *key, long *numval, char **stringval, int opt_flags, int opt_type,
```

Comment:
This is *very* similar to `set_option_value_for`. Is there any reasonable way we can unify them?

Reply:
Imo the effort to design a common abstraction for just these two functions is not worth it.

Reply:
Design? Come-on, it isn't that difficult. It's just basic refactoring.